windows/CodePushUpdateUtils.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace CodePush.ReactNative
+{
+    class CodePushUpdateUtils
+    {
+        // TODO: Generate binary hash
+        // private static readonly String CODE_PUSH_HASH_FILE_NAME = "CodePushHash.json";
+
+        public async static Task CopyNecessaryFilesFromCurrentPackage(StorageFile diffManifestFile, StorageFolder currentPackageFolder, StorageFolder newPackageFolder)
```

Comment:
this does more than just copying...it also deletes the files that were deleted by the update. this is more like `performDiffWithPackage` or something